lib/phoenix_live_view/channel.ex

```diff
@@ -62,6 +82,12 @@ defmodule Phoenix.LiveView.Channel do
     e -> reraise(e, __STACKTRACE__)
   end
 
+  def handle_info({:EXIT, _pid, _reason} = msg, state) do
+    msg
+    |> view_handle_info(state.socket)
+    |> handle_result({:handle_info, 2, nil}, state)
+  end
```

Comment:
We can fully remove this clause i think.

Reply:
We need it if folks are trapping exits in their LV

Reply:
(unrelated to this PR)

Reply:
ah the regular handle_info fallthrough will forward these already. Good call!